src/dsl/word/Once.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2013      Trent Houliston <trent@houliston.me>, Jake Woods <jake.f.woods@gmail.com>
+ *               2014-2017 Trent Houliston <trent@houliston.me>
+ *               2021      Trent Houliston <trent@houliston.me>
```

Comment:
Shouldn't this include @WongAaronSW

Reply:
According to @TrentHouliston,  it's reserved for people who maintain the code, not for "Authors"'.